File: Programy c/l2/zad3i2.c
#include <stdio.h>
#include <math.h>

int main()//kompiluj z -lm
{

double srt=0.001; 
double result=1;

for(int i=1; i<1001; i++)
{
    result = result*pow(i,srt);
}

printf("approx: %lf\n\n",result);

return 0;
}

File: Programy py/zad3i2.py
import math

result, degree = 1.0, float(input("factorial of: "))

for i in range (1,int(degree)+1):
    result = result*pow(float(i),(1/degree))

print(str(int(degree))+"th root of "+str(int(degree))+"! is "+str(result))